l3.c

```diff
@@ -29,48 +46,62 @@ struct l3_log
     struct l3_entry slots[L3_MAX_SLOTS];
 } *l3_log;
 
+// ****************************************************************************
 int
 l3_init(const char *path)
 {
     int fd = -1;
     if (path)
     {
         fd = open(path, O_RDWR | O_CREAT, 0666);
-        if (fd == -1) return -1;
+        if (fd == -1) {
+            return -1;
+        }
```

Comment:
I **strongly** encourage and request that we should adhere to the practice of enclosing even single-line `if()`, `while()` bodies inside `{ }`and conform to proper indentation on the next line.

I have applied this change to few files.

Long-term: We should consider standardizing on a `clang-format` format-file, run that in CI, so all new code comes out looking consistent.